code/modules/mining/laborcamp/laborstacker.dm

```diff
@@ -129,10 +129,15 @@ GLOBAL_LIST(labor_sheet_values)
 	if(stacking_machine)
 		stacking_machine.labor_console = src
 
-/obj/machinery/mineral/labor_claim_console/emag_act(mob/user)
+/obj/machinery/mineral/labor_claim_console/emag_act(mob/user, obj/item/card/emag/emag_card)
 	if(!(obj_flags & EMAGGED))
 		obj_flags |= EMAGGED
-		to_chat(user, span_warning("PZZTTPFFFT"))
+		if (user)
+			balloon_alert(user, "id authenticator short-circuited")
+		visible_message(span_warning("[src] lets out a few sparks!"))
+		do_sparks(2, TRUE, src)
```

Comment:
Soul removal 😔 
As frequently otherwise stated, if you have other visual feedback (sparks) and it's not providing additional context, you probably don't need a balloon alert. I guess this one might be fine in that it tells you what you actually did.